_providers/mail2tor.md

```diff
@@ -0,0 +1,23 @@
+---
+name: mail2tor
+status: PREPARATION
+domains:
+- mail2tor.com
+strict_tls: true
+server:
+- type: imap
+  hostname: g77kjrad6bafzzyldqvffq6kxlsgphcygptxhnn4xlnktfgaqshilmyd.onion
+  port: 143
+  socket: PLAIN
```

Comment:
the script `src/provider/update.py` crating `src/provider/data.rs` currently does not support `PLAIN`, i already have a fix for that, will do a pr.

Reply:
pr: https://github.com/deltachat/deltachat-core-rust/pull/2604

Reply:
as this one is merged... do we need to wait for a core release or can we merge this?

Reply:
no need to wait, at least not for that reason :)